File: test_files/typescript/generated_autocomplete_cases.ts
export function formatStatus(name: string, active: boolean): string {
  return `${name}:${active ? "active" : "inactive"}`;
}

export function caseCallCompletion(user: string): string {
  const status = formatStatus(
  return status;
}

export function caseSuffixOnly(items: string[]): string {
  const first = items[0] ?? "none";
  return first.toLower
}

File: test_files/python/generated_autocomplete_cases.py
"""Generated Python autocomplete fixtures for gap-fill manual tests."""

from typing import Iterable


def format_status(name: str, active: bool) -> str:
    return f"{name}:{'active' if active else 'inactive'}"


def case_call_completion(user: str) -> str:
    status = format_status(
    return status


def case_suffix_only(values: Iterable[str]) -> str:
    first = next(iter(values), "none")
    return first.up
